test/python/default_richards_wells.py

```diff
@@ -6,11 +6,11 @@
 
 from parflow import Run
 from parflow.tools.fs import mkdir, get_absolute_path
+from utils import py_pftestFile
 
-drich = Run("default_richards_wells", __file__)
-
+run_name = "default_richards_wells"
```

Comment:
Is this easier?   Saves having to change the runname all the time?  Not super sure, might be less clear.

import pathlib
run_name = pathlib.Path(__file__).stem

Reply:
The test files are already written, so I'm not sure if that saves time. Writing the run_name explicitly might be clearer too.